gninasrc/lib/flexinfo.cpp

```diff
@@ -103,6 +130,43 @@ void FlexInfo::sanitizeResidues(OpenBabel::OBMol& receptor) {
   }
 }
 
+void FlexInfo::checkResidue(OpenBabel::OBResidue* residue)
+{
+  using namespace OpenBabel;
+
+  double n_atoms_with_buffer = 0.0;
```

Comment:
Just for the sake of defensive programming, let's immediately return if residue == nullptr here